docs/manual/docs/examples/spo/adding-app-catalog.md

```diff
@@ -0,0 +1,18 @@
+# Hide SharePoint list from Site Contents
+
+Author: [David Ramalho](https://sharepoint-tricks.com/tenant-app-catalog-vs-site-collection-app-catalog/)
+
+
+When you just want to deploy certain SharePoint solution to a specific site, it's required to create an app catalog for that site, the below script will create it for the site. On the article link above you can check where you can use App catalog for the site instead of global app catalog.
+
+```powershell tab="PowerShell Core"
+
+spo login https://contoso-admin.sharepoint.com
```

Comment:
`spo login <url>` is an old command that's been replaced by `o365 login`. Would you mind updating the script to reflect this change? 😊

Reply:
Changed to the new command, here and on my blog 😜